.github/workflows/test.yml

```diff
@@ -90,3 +90,18 @@ jobs:
       - uses: actions/checkout@v4
       - uses: streetsidesoftware/cspell-action@v6
 
+  check_rust_backward_compatibility_1_81:
+    needs: cargo-fmt
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: 1.81
+        default: true
+    - name: Install libudev (Linux only)
+      run: sudo apt-get update && sudo apt-get -y install libudev-dev libsystemd-dev
```

Comment:
it looks like https://github.com/near/near-api-rs/blob/main/.github/workflows/release-plz.yml#L25 is still lacking the same `sudo apt-get update &&` string prefix